template/content/app.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var fetchConfig = require('zero-config');
+
+var ApplicationClients = require('./clients.js');
+var Endpoints = require('./endpoints.js');
+
+module.exports = Application;
+
+function Application(options) {
+    if (!(this instanceof Application)) {
+        return new Application(options);
+    }
+
+    var self = this;
+    options = options || {};
+
+    self.config = fetchConfig(__dirname, {
+        dcValue: 'todo',
+        seed: options.seedConfig,
+        loose: false
+    });
+
+    self.clients = ApplicationClients(self.config, {
+        logger: options.logger,
+        statsd: options.statsd
+    });
+    self.endpoints = Endpoints(self);
```

Comment:
honestly don't like this pattern, endpoints shouldn't need the whole app object, just pass it what it needs, preferably the channel and the thrift 'codec'